public/js/timeline.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
```

Comment:
@Ezlo123 when a file is tracked just adding it to the ignore list doesn't help, that just prevents new files following those rules to be tracked. Use "git rm --cached [filename]" to stop tracking the file (it must be specified in the gitignore too, but it is i guess)